fuel-tx/src/transaction/types/input/message.rs

```diff
@@ -143,15 +183,20 @@ pub mod specifications {
 #[derive(Default, Derivative, Clone, PartialEq, Eq, Hash)]
 #[derivative(Debug)]
 #[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+#[cfg_attr(
+    feature = "da-compression",
+    derive(fuel_compression::Compress, fuel_compression::Decompress)
```

Comment:
I think we need tp implement `DecompressibleBy` via `DecompressionContext` manually since on the decompressor side we need to fetch skipped fields

Reply:
9b33143